.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Default all text files to use LF line endings
+* text eol=lf
+
+visualization/references/*.{txt,tex} text=auto
```

Comment:
I _think_ it's `eol=auto` rather than `text=auto`?  The `txt` and `tex` formats should be detected as `text`, and it's the EOL we care about.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/13928/commits/d65c3e9ca624dd7c03d1f92fe1eb5cfc62e2fad8